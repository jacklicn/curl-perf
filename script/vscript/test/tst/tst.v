var a,b,c,d,e

a=1
b=2
c=3
d=4

e = (a + b) + c * d 

print(e)

sub foo( a, b )
 
  print( "proc foo" )
  print( a )
  print( b )

end

foo( (a * b) + (c - d), e )



