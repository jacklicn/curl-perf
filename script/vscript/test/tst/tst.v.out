*** syntax ok ***
*** semantics and code generation ok ***

>15

>proc foo

>1

>15
